Header header
string marker_id
float64 x
float64 y
sensor_msgs/CameraInfo camera_info
